README.md

```diff
@@ -762,10 +763,10 @@ func printSomeFile() {
 
 Instead, we should be using Swift's `try`/`catch` behavior when it is appropriate to know the reason for the failure.
 
-Use a `struct` such as the following:
+You can use a `struct` such as the following:
 
 ```swift
-struct Error: ErrorType {
+struct Error: Swift.Error {
```

Comment:
I feel like you should name this differently instead of `Error`. Use like `MyError`